app/controllers/activities_controller.rb

```diff
@@ -0,0 +1,34 @@
+class ActivitiesController < ApplicationController
+  def index
+    @category = Group.find_by(id: params[:group_id])
+    @transactions = Activity.includes(:groups).where("groups.id = #{params[:group_id]}").references(:groups)
+    @total_amount = 0
+    @transactions.each do |activity|
+      @total_amount += activity.amount
+    end
+  end
+
+  def new
+    @activity = Activity.new
+    @groups_list = Group.where(user_id: current_user.id)
+  end
```

Comment:
- [ ] We don't implement SQL Queries inside the controller, we use the model to keep the SQL queries implementation.